.eslintrc.json

```
@@ -38,6 +39,10 @@
 				},
 				"groups": [ "builtin", "external", "internal", "parent", "sibling", "index", "type" ]
 			}
-		]
+		],
+		"mobx/exhaustive-make-observable": "warn",
+		"mobx/unconditional-make-observable": "error",
+		"mobx/missing-make-observable": "error",
+		"mobx/missing-observer": "off"
```

```suggestion
		"mobx/missing-observer": "warn"
```
In a real-life scenario, this rule should be enabled. I kept it off for this PR since it adds a lot of noise (it prompts us to wrap _every single component_ in an `observer` call)

---

As for me, if we go with MobX, I would advocate avoiding any custom rule unless we have strong arguments to override the default behavior.